source/beamline/AMPVControl.cpp

```diff
@@ -249,6 +250,13 @@ AMControl::FailureExplanation AMPVControl::move(double setpoint) {
 			return NotConnectedFailure;
 		}
 		setpoint_ = setpoint;
+
+		// Update the move progress values.
+
+		updateMoveStart();
```

Comment:
Since all three of these functions are typically called together this way, maybe make a convenience method that calls all three when it is appropriate to do so?